hw5.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+$n = 10;
+
+function Fibonachi($n)
+{
+    $Sum = 0;
+    $a = array();
+    $a [0] = 0;
+    $a [1] = 1;
+    if ($n == 1) 
+    {
+        $Sum = $a [0] = 0;
```

Comment:
тут мені здається використання $a [0]  є зайвим
```suggestion
        $Sum = 0;
```